src/Lib/Managers/SelectedBoundManager.vala

```diff
@@ -325,7 +337,11 @@ public class Akira.Lib.Managers.SelectedBoundManager : Object {
      * Move the item based on the mouse click and drag event.
      */
     private void move_from_event (
-        Lib.Items.CanvasItem item, double event_x, double event_y) {
+        Lib.Items.CanvasItem item,
+        double event_x,
+        double event_y
+        ) {
```

Comment:
We can inline this with `private void move_from_event (Items.CanvasItem item, double event_x, double event_y) {`